README.md

```diff
@@ -20,35 +20,46 @@ Use node version `14.16.0` to run the app. You can use [nvm](https://github.com/
 
 With nvm installed:
 
-`nvm install 14.16.0`
+`nvm install 14.21.1`
 
 ## Setup
 
 ### Install dependencies
+
 `yarn`
 
 ## Development server
 
 Serve local files with hot reload at localhost:3000
 
 ### Development API
-`yarn start`
+
+`yarn start-local`
+
+You will need to copy the oauth property from src/site-config/dev.json to src/site-config/site.json in order to render the application locally.
```

Comment:
This has bitten me too and I've had to follow the same steps to get a local setup running. LGTM

Reply:
There are 3 .json files in `src/site-config`, which I believe are intended to get copied to `site.json` -- depending on which startup is asked for: local, dev, or prod. I think the local.json has not been kept up-to-date, and should be modified to include all the necessary settings to run. I also think some of the motivation for the local.json was to access a locally-running Pennsieve API.